docs/content/developers/building-contributing.md

```diff
@@ -24,70 +24,45 @@ There are several ways to use DDEV’s latest-committed HEAD version:
 
 Each [PR build](https://github.com/ddev/ddev/actions/workflows/pr-build.yml) creates GitHub artifacts you can use for testing, so you can download the one you need from the PR page, install it locally, and test using that build.
 
-Download and unzip the appropriate binary and place it in your `$PATH`.
+!!!tip "You can also [downgrade to an older version of DDEV](../users/usage/faq.md#how-can-i-install-a-specific-version-of-ddev) (perform a rollback)."
 
-## Rollback to a previous version
+Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. Since not every OS has `$HOME/bin` in `$PATH`, you can create the folder and add it to your path by updating `~/.bashrc`, `~/.zshrc`, or another relevant shell configuration file with these commands:
 
-You can also [downgrade to an older version of DDEV](../users/usage/faq.md#how-can-i-install-a-specific-version-of-ddev).
-
-### Homebrew with macOS or Linux
-
-If you’re using Homebrew, start by unlinking your current binary:
-
-```
-brew unlink ddev
+```bash
+mkdir ~/bin
+echo 'export PATH="$HOME/bin:$PATH"' >>~/.bashrc
+source ~/.bashrc
+# Verify that `$HOME/bin` is the first entry in your `$PATH`
+echo $PATH
 ```
 
-Next, unzip the binary you downloaded, make it executable, and move it to your bin folder:
+Choose a ZIP file for your OS and architecture, download it using `wget`, `curl`, `dl` or using a browser and unzip it, make it executable, and move it to the `~/bin` folder:
 
-```
-unzip ddev.zip
-chmod +x ddev && sudo mv ddev /usr/local/bin/ddev
+```bash
+# Example for macOS Apple Silicon:
+unzip ddev-macos-arm64.zip
+chmod +x ddev && mv ddev ~/bin/ddev
+rm ddev-macos-arm64.zip
 ```
 
-Verify the replacement worked by running `ddev -v`. The output should be something like `ddev version v1.22.5-alpha1-70-g0852fc2df`, instead of the regular `ddev version v1.22.5`.
+![Github Action PR Comment ZIP files](../images/github-action-pr-comment.png)
```

Comment:
Could a link to the nightly builds be added, in case someone wants to try out the "Testing a PR" process, and just needs a quick zip.file? I could easily see myself need this, at a later point in time, and not have to dig through PR's, to find a list of artifacts would be nice ...

```suggestion
![Github Action PR Comment ZIP files](../images/github-action-pr-comment.png)

Tip: If you need a zip-file to try out the "Testing a PR" process, see the [nightly builds](https://nightly.link/ddev/ddev/workflows/master-build/master).
```